PROJECT_LOG.md

```
@@ -5,6 +5,13 @@ This is the operational logbook, not the release changelog (see [docs/changelog.
 
 ---
 
+## 2026-06-08
+
+### PR #1213 merged (`79138b0`) — docs(mobile-catalog): UML conception study for the mobile beer catalogue (UC1/2/3)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)